File: geek-university-algoritmo-e-logica-de-programacao/python/exercicio_20.py
valor_hora = 10.00
valor_hora_excedente = 20.00
extra = 0

codigo = int(input("Informe o codigo: "))
qtd_horas = float(input("Informe a quantidade de horas trabalhadas: "))

if qtd_horas > 50:
    extra = (qtd_horas - 50) * valor_hora_excedente
    salario = (50 * valor_hora) + extra
    print("Salario Total R$ {0:.2f}".format(salario))
    print("Salario excedente R$  {0:.2f}".format(extra))
else:
    salario = (qtd_horas * valor_hora)
    print("Salario Total R$  {0:.2f}".format(salario))
    print("Salario excedente R$ {0:.2f}".format(extra))

File: geek-university/algoritmo-e-logica-de-programacao/java/exercicio_20.java
import java.util.Scanner;

public class exercicio_20 {

    public static void main(String[] args) {
        int codigo;
        float qtd_horas, e = 0, salario, valor_hora = (float) 10.00;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe o codigo: ");
        codigo = teclado.nextInt();
        System.out.print("Informe a quantidade de horas trabalhadas: ");
        qtd_horas = teclado.nextFloat();

        if (qtd_horas > 50) {
            e = (float) ((qtd_horas - 50) * 20.00);
            salario = (50 * valor_hora) + e;
            System.out.printf("Salario Total R$ %.2f\n", salario);
            System.out.printf("Salario excedente R$ %.2f\n", e);
        } else {
            salario = qtd_horas * valor_hora;
            System.out.printf("Salario Total R$ %.2f\n", salario);
            System.out.printf("Salario excedente R$ %.2f\n", e);
        }

        teclado.close();
    }

}
